spec/01_tic_tac_toe_spec.rb

```diff
@@ -52,11 +52,29 @@
       end
     end
 
+    describe '#input_to_index' do
+      it "accepts the user's input (a string) as an argument" do
+        game = TicTacToe.new
+        expect{game.input_to_index}.to raise_error(ArgumentError)
```

Comment:
@aturkewi yeah, http://stackoverflow.com/questions/21567838/when-to-use-curly-braces-vs-parenthesis-in-expect-rspec-method#21568225